yt_dlp/extractor/bilibili.py

```diff
@@ -1025,6 +1060,64 @@ def _extract_video_metadata(self, url, video_id, season_id):
                 'description': self._html_search_meta('og:description', webpage)
             })
 
+    def _get_comments_reply(self, root_id, next_id=0, display_id=None):
+        comment_api_raw_data = self._download_json(
+            'https://api.bilibili.tv/reply/web/detail', display_id,
+            note=f'Downloading reply comment of {root_id}',
+            query={
+                'platform': 'web',
+                'ps': 3,  # comment's reply per page (default: 3)
```

Comment:
can we increase this?

Reply:
In https://github.com/yt-dlp/yt-dlp/pull/6079/commits/28cc5419b7b87db9cf5368315f073e073e44c1b7, i've increased to 20